js/renderer.js

```diff
@@ -749,8 +766,8 @@ function newImage(sender, type, newImageArray) {
     timer: 5000,
     icon: "success"
   }).then((value) => {
-    currentImageIndex = images.length;
-    loadImage(true, 0);
+    currentImageIndex = 0;
+    loadImage(false, 0, true);
```

Comment:
please verify this is ok. Need to jump directly to a new picture, as the other logic won't work in random order.

Reply:
I think it has the side-effect that when new pictures come in while in pause, the picture changes to the latest one? (To be investigated)